### DISCUSSION TITLE
Real-Time Feedback on Changes in Shared Projects.

### DISCUSSION THREAD

**Discussion by mahmoodi-source** [opengisch/QField#5763] [Q&A]
Hi,
I’m wondering if QField has a feature that provides real-time feedback on changes made to a project by other team members. Specifically, is there a way to receive notifications or alerts when a group member makes an update or change to the project while working in the field? This would help improve collaboration and keep everyone on the same page during fieldwork.
Thanks for any information!

**Comment by thamishan** [comment]
I'm looking for this same feature as well! We have a team of ecologists registering in different maps every night. It's really hard to keep track of changes as admin.
